apps/website/src/components/chart/utils/get-percentage.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * An util function that calculates the percentage of a value from a total
+ * @param value - the value to calculate the percentage
+ * @param total - the total value
+ **/
+
+export const calculatePercentage = (value: number, total: number): number =>
```

Comment:
file is called `get-percentage` whereas the function is `calculatePercentage`

Reply:
I totally forgot to change the name of the function. Good catch.